integrations/kube-agent-updater/pkg/controller/suite_test.go

```diff
@@ -17,74 +17,9 @@ limitations under the License.
 package controller
 
 import (
-	"context"
-	"testing"
-
-	"github.com/stretchr/testify/require"
 	"golang.org/x/exp/rand"
-	appsv1 "k8s.io/api/apps/v1"
-	core "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
-	corescheme "k8s.io/client-go/kubernetes/scheme"
-	ctrl "sigs.k8s.io/controller-runtime"
-	kclient "sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/envtest"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
-	runtimescheme "sigs.k8s.io/controller-runtime/pkg/scheme"
-)
-
-var (
-	SchemeBuilder = &runtimescheme.Builder{GroupVersion: appsv1.SchemeGroupVersion}
-	scheme        = runtime.NewScheme()
 )
 
-func init() {
```

Comment:
What happened to all this stuff? Was this a leftover?

Reply:
Yes, those functions were helpers for the first end-to-end testing approach. However, I had to change the testing technique mid-course because we rely on pod status for some maintenance triggers and controllers and `envtest` runs a real Kubernetes API server which starts to fight for pod status reconciliation.